src/Glpi/Form/Destination/CommonITILField/ContentField.php

```diff
@@ -86,10 +86,11 @@ public function renderConfigForm(
                 value,
                 label,
                 options|merge({
-                    'enable_richtext': true,
-                    'enable_images': false,
-                    'enable_form_tags': true,
-                    'form_tags_form_id': form_id
+                    'enable_richtext'  : true,
+                    'enable_images'    : false,
+                    'enable_form_tags' : true,
+                    'form_tags_form_id': form_id,
+                    'content_style'    : 'body { line-height: 2.3; } h2 { margin: 1rem 0 0.5rem; }',
```

Comment:
Could this be changed globally? Maybe by changing the `body.mce-content-body` style.

Reply:
The style changes for headings have been added to the global style file for the rich text editor. However, the `line-height` directive can't be general and would cause line spacing to be far too wide. The directive is required here to prevent tags from overlapping.